woocommerce/woocommerce-paypal-payments
# Add woocommerce inbox variant

Add WooCommerce inbox variant option.

#Changes proposed in this Pull Request:

P2: paJDYF-1uJ-p2
Fixes https://github.com/Automattic/woocommerce.com/issues/10117

Description: This PR adds a new option `woocommerce_inbox_variant_assignment ` for the Remote Inbox Notification on plugin activation or upgrade.

* This PR does not clean up `woocommerce_inbox_variant_assignment ` option on uninstallation so that we don't create a different value in case user decides to install the plugin again.  

* I'm calling `woocommerce_inbox_variant_assignment` method in the `plugins_loaded` hook since I wasn't able to find a way to trigger a method when the plugin gets upgraded. It's not ideal, but performance should be ok since the option is autoloaded. 


# Testing instructions

**Testing Activation**
1. Ensure `woocommerce_inbox_variant_assignment` option does not exist. If it does, delete it before testing.
2. Deactivate the plugin then activate it again
3. `woocommerce_inbox_variant_assignment` option should be created.
4. Repeat step 2 and ensure that the value of `woocommerce_inbox_variant_assignment` does not change.
